packaging/install.sh

```diff
@@ -5,9 +5,10 @@ echo "Installing cloudify manager installer..."
 echo "###################################################################"
 
 # Using $SUDO_USER instead of $USER here because fpm runs the script as sudo
-sudo chown $SUDO_USER:$SUDO_USER -R /opt/cloudify-manager-install
+# and needs access to the config file as the user
+sudo chown $SUDO_USER:$SUDO_USER /opt/cloudify/config.yaml
```

Comment:
Have you made sure that access to the config file is the only thing necessary here? Access to the rest of the directory isn't?

Reply:
I made sure insofar as I ran the install process and it worked- this is also replicating the permissions that were previously there (as only that file was in that directory after the single tar removal), so it would've been odd if it had managed to obtain extra requirements just by having the file moved (odd, not impossible, but the manual runs of the tool and following install worked).

Reply:
Great. Good enough for me